docker-compose.yaml

```diff
@@ -1,26 +1,58 @@
-version: '3.9.1'
+version: "3.9.1"
 
-# You must manually run `ankerctl config import` atleast once
-# docker run \
-# -v ankerctl_vol:/root/.config/ankerctl \
-# -v "$HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json" \
-# ankerctl config import /tmp/login.json
 services:
-  ankerctl:
-    image: ankerctl/ankerctl:latest
-    container_name: ankerctl
-    restart: unless-stopped
-    build: .
-    environment:
-      - FLASK_PORT=4470
-      - FLASK_HOST=0.0.0.0
-    volumes:
-      - ankerctl_vol:/root/.config/ankerctl
-    ports:
-      - 127.0.0.1:4470:4470
-    entrypoint: "/app/ankerctl.py"
-    command: ["webserver", "run"]
+    ankerctl_seed_mac:
+        image: busybox
+        container_name: ankerctl_seed_mac
+        restart: "no"
+        volumes:
+            - ankerctl_vol:/root/.config/ankerctl
+            - "$HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json"
+        command: sh -c "cp /tmp/login.json /root/.config/ankerctl/login.json || true"
+    ankerctl_seed_linux:
+        image: busybox
+        container_name: ankerctl_seed_linux
+        restart: "no"
+        volumes:
+            - ankerctl_vol:/root/.config/ankerctl
+            # TODO: verify this is the correct linux path
+            - "$HOME/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json"
+        command: sh -c "cp /tmp/login.json /root/.config/ankerctl/login.json || true"
+    ankerctl_import:
+        # image: ghcr.io/ankermgmt/ankermake-m5-protocol:latest
+        image: ankerctl/ankerctl:latest
+        container_name: ankerctl_import
+        restart: "no"
+        build: .
+        volumes:
+            - ankerctl_vol:/root/.config/ankerctl
+        entrypoint: "/app/ankerctl.py"
+        command: ["config", "import", "/root/.config/ankerctl/login.json"]
+        depends_on:
+            - ankerctl_seed_mac
+            - ankerctl_seed_linux
+        
+    ankerctl:
+        # image: ghcr.io/ankermgmt/ankermake-m5-protocol:latest
+        image: ankerctl/ankerctl:latest
```

Comment:
Can you explain how `ankerctl/ankerctl:latest` works? I'm not quite familiar with that syntax.

Suppose we make sure `ghcr.io/ankermgmt/ankermake-m5-protocol:latest` always has a fresh image available. What are the pros/cons of using that image here, vs pointing to `ankerctl/ankerctl`, which (I imagine) points to the locally-built image?

Reply:
Eventually `ghcr.io/ankermgmt/ankermake-m5-protocol:latest` will be preferred, however for testing it won't work since I need the latest docker container available on my machine. 

We can switch ankerctl/ankerctl to ghcr once we have the updated web gui published. 
I can actually change this in git, and just override it temporarily while testing.